kolibri/core/device/permissions.py

```diff
@@ -47,3 +47,18 @@ def has_permission(self, request, view):
 class IsSuperuser(DenyAll):
     def has_permission(self, request, view):
         return request.user.is_superuser
+
+
+class LODUserHasSyncPermissions(BasePermission):
+    """
+    When a device for LOD is provsioned, after the first user has been synced,
+    this user must be able to sync more users
+    """
```

Comment:
I see a lot of `LOD` references, which I assume stands for the user-facing concept we settled on, "Learn-only device"? I would personally recommend spelling out `learn_only_device` for clarity rather than using an acronym, even though it's more verbose.

In other places in this PR we also use `subset_of_users_device` and even `soud` so it would be good to tighten up our naming consistency.

Reply:
I think for dev facing code, I'd prefer we stick to `subset_of_users_device` which is then consistent with the previous references we have made to this. We can still call it a `Learner only device` in user facing strings.